### PR TITLE
Databases/week1 homework

### PR DESCRIPTION
**Pull request by Gohar33**

### CODE REVIEW

File: databases/week1/Week1.sql [hunk 1]
```diff
@@ -0,0 +1,67 @@
+-- 1. Find out how many tasks are in the task table*/
+SELECT COUNT(*)
+FROM task;
+-- 35
```
Comment (Elia-Git) [inline databases/week1/Week1.sql:4]: In line 4 remove the commented lines. This applies to the whole work. You don't even need to provide the answers!

**Review by Elia-Git — COMMENTED** [review]
Good work Gohar! Two recommendations ..
1. It is not a good practice to include any commented line in your code. So remove all commented lines.
2.  Avoid using * everywhere. Use it if you really need it. Otherwise, it could disturb your work. You can select few rows instead. 

Let me see your changes and we can ship it.

File: databases/week1/Week1.sql (hunk 1 above)
Comment (Gohar33) [reply databases/week1/Week1.sql:4]: Hi Elias, thanks a lot for your feedback. We were told to include comments so mentors can check them easily and I thought it concern answers also. Will take it out now and will change * to a specific selection.

Comment (Elia-Git) [reply databases/week1/Week1.sql]: I think it must be a mistake that you have been told to push commented lines with your code.

**Review by Elia-Git — APPROVED** [review]
Thanks for the quick fix. Ship it.

File: databases/week1/Week1.sql (hunk 1 above)
Comment (martamatos) [reply databases/week1/Week1.sql]: I think this is my bad for not telling it to Elias, we actually asked the students to write either the exercise number or the full description before each query to be easier to correct. Sorry for the confusion :/

**Pull request merged**